src/frontend/komponenter/Fagsak/Vedtak/KorrigerEtterbetalingModal/KorrigerEtterbetalingModal.tsx

```diff
@@ -109,16 +105,26 @@ const KorrigerEtterbetalingModal: React.FC<IKorrigerEtterbetalingModal> = ({
         onClose();
     };
 
+    const hentLabelForÅrsak = (årsakValue: string) =>
+        årsaker.find(årsak => årsak.value === årsakValue)?.label;
+
     return (
         <Modal open={visModal} onClose={lukkModal}>
             <StyledModalContent>
-                <StyledModalHeader size="medium">Korriger etterbetaling</StyledModalHeader>
+                <StyledModalHeader size="medium" level={'2'}>
+                    Korriger etterbetaling
+                </StyledModalHeader>
                 <StyledSkjema feil={false}>
                     <div>
-                        <StyledSelect
+                        <StyledFamilieSelect
                             label={'Årsak'}
                             id={'korrigering-aarsak'}
                             value={skjema.felter.årsak.verdi}
+                            lesevisningVerdi={
+                                skjema.felter.årsak.verdi === ''
+                                    ? 'Ingen årsak valgt.'
+                                    : hentLabelForÅrsak(skjema.felter.årsak.verdi)
```

Comment:
Dette er for å unngå at enumverdien vises i lesevisning
<img width="564" alt="Screenshot 2022-10-20 at 10 51 53" src="https://user-images.githubusercontent.com/2379098/196927295-aca9a98a-656a-40cd-aaaa-45d72706c1ba.png">